packages/file_selector/file_selector_android/android/build.gradle

```diff
@@ -0,0 +1,61 @@
+group 'io.flutter.plugins.file_selector_android'
+version '1.0'
+
+buildscript {
+    repositories {
+        google()
+        mavenCentral()
+    }
+
+    dependencies {
+        classpath 'com.android.tools.build:gradle:7.3.0'
+    }
+}
+
+rootProject.allprojects {
+    repositories {
+        google()
+        mavenCentral()
+    }
+}
+
+apply plugin: 'com.android.library'
+
+android {
+    namespace 'io.flutter.plugins.file_selector_android'
```

Comment:
@stuartmorgan Is there a preference for the org of new Android implementations? This is the standard, but I think new ones should be `dev.flutter.packages.<pluginname>`.

Reply:
Yes, we should definitely use dev going forward. A while ago I fixed all the legacy references to io.flutter in the plugins repo, except for Android because changing namespaces is non-trivial so I punted on that part.